big-practice/src/index.html

```diff
@@ -0,0 +1,93 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Book Managerment</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="./style/main.css">
+</head>
+
+<body>
+  <header class="page-header">
+    <div class="container-xxl">
+      <h1>
+        <a href="javascript:void(0)" class="header__logo">
+          <img src="./assets/images/logo.svg" alt="Logo" class="logo__image">
+          <span class="logo__text">Reading Book</span>
+        </a>
+      </h1>
+    </div>
+  </header>
+  <main class="page-main">
+    <section class="section-layout-2-col">
+      <div class="container-xxl">
+        <div class="wrap">
+          <aside class="layout-col-left">
+          </aside>
+          <div class="layout-col-main">
+            <a href="javascript:void(0)" class="creating-logo">
+              <img src="./assets/images/create-logo.svg" alt="Create Logo">
+              <span>Create book</span>
+            </a>
+            <div class="table-responsive">
+              <div class="table-wrapper">
+                <table class="table table-hover table-striped">
+                  <thead class="table__header">
+                    <tr>
+                      <th>#</th>
+                      <th>Cover Image</th>
+                      <th>Book Name</th>
+                      <th>Author</th>
+                      <th>Edit</th>
+                      <th>Delete</th>
+                    </tr>
+                  </thead>
+                  <tbody id="book-list">
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Modal -->
+    <div class="modal fade" id="delete-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel"
+      aria-hidden="true">
+      <div class="modal-dialog modal-dialog-delete" role="document">
+        <div class="modal-content modal-content--delete">
+          <div class="modal-header modal-header--delete">
+            <h5 class="modal-title modal-title--delete" id="exampleModalLabel">Delete book</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body modal-body--delete">
+            <p>Are you sure you want to delete this book?</p>
+          </div>
+          <div class="modal-footer modal-footer--delete">
+            <button type="button" class="btn btn-secondary btn-primary--delete" data-dismiss="modal" id="btn-model-cancel">Cancel</button>
+            <button type="button" class="btn btn-primary btn-secondary--delete" data-dismiss="modal" id="btn-modal-delete">Delete</button>
```

Comment:
Class names will cause confusion.
  - btn btn-**secondary** btn-**primary**--delete , 
  - btn btn-**primary** btn-**secondary**--delete
  - btn-model-cancel =>>> modal -> not model